<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysSetForm</class>
 <widget class="QWidget" name="SysSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../img.qrc">
    <normaloff>:/img/images/monSet.png</normaloff>:/img/images/monSet.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>635</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>主机管理</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>618</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>屏幕等待</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>系统时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确 定</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_reset">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重 置</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCSTime">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>年</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtYear">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMonth">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>月</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>日</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDay">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSec">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtHour">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMinute">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>618</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>数据清除</string>
    </property>
    <widget class="QPushButton" name="btn_clear_warn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除报警数据</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear_opt">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除操作数据</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calibration">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重新校准屏幕</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear_err">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除故障数据</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>618</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>系统配置</string>
    </property>
    <widget class="QLineEdit" name="txtNet0">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNet1">
     <property name="geometry">
      <rect>
       <x>316</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>网络1节点数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>网络0节点数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTimer">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重发次数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSys1Reset">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重 置</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSys1OK">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确 定</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>568</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(0~5)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(0~128)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>376</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(0~128)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart1_badurate_comboBox">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>27</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart1_badurate_label">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>25</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>串口1：波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart1_parity_comboBox">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>27</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart1_parity_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>25</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart1_data_comboBox">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>27</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart1_data_label">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>25</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart1_stop_comboBox">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>27</y>
       <width>70</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart1_stop_label">
     <property name="geometry">
      <rect>
       <x>486</x>
       <y>25</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart2_badurate_comboBox">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>62</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="usart2_parity_comboBox">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>62</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="usart2_stop_comboBox">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>62</y>
       <width>70</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart2_badurate_label">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>60</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>串口2：波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart2_data_comboBox">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>62</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart2_data_label">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="usart2_parity_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
    </widget>
    <widget class="QLabel" name="usart2_stop_label">
     <property name="geometry">
      <rect>
       <x>486</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QLabel" name="usart3_stop_label">
     <property name="geometry">
      <rect>
       <x>486</x>
       <y>95</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart3_stop_comboBox">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>97</y>
       <width>70</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart3_data_label">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>95</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="usart3_parity_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>95</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usart3_badurate_comboBox">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>97</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="usart3_parity_comboBox">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>97</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="usart3_data_comboBox">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>97</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="usart3_badurate_label">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>95</y>
       <width>108</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>串口3：波特率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="usart_btn_close">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>设置串口</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(1~247)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="local_slave_address_txt">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>本地从机地址</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../img.qrc"/>
 </resources>
 <connections/>
</ui>
